app/views/drivers/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<h2> Driver details page </h2>
+<p></p>
+<h3>This is <%= @driver.name %> driver page</h3>
+<p>Driver's name: <%= @driver.name %></p>
+<p>VIN: <%= @driver.vin %></p>
+<p>Available: <%= @driver.available %></p>
+<p>Driver's average rating: <%=@driver.average_rating %></p>
+<p>Total revenue: <%=dollar_format(@driver.earned)%></p>
+<%= link_to 'Edit', edit_driver_path(@driver) %>
+<%= link_to 'Delete', @driver, method: :delete, data: { confirm: 'Are you sure?' } %>
+
+<table id='trips-table' class="table table-hover">
```

Comment:
You can convert this content to a partial.  Then you can display the same table anywhere you list trips.